e2e-tests/insurance/cypress/e2e/journeys/your-buyer/alternative-currency/alternative-currency-page.spec.js

```diff
@@ -94,23 +99,77 @@ context('Insurance - Your Buyer - Alternative currency - As an exporter, I want
   });
 
   describe('form submission', () => {
-    describe('when submitting an empty form', () => {
+    describe('when submitting a form when leaving alternative currency input blank', () => {
       beforeEach(() => {
         cy.navigateToUrl(url);
       });
 
       it('should render validation errors', () => {
-        cy.clickSubmitButton();
+        rendersAlternativeCurrencyValidationError();
+      });
+    });
+
+    describe('when submitting a fully filmed form', () => {
```

Comment:
filmed? Could we be more descriptive e.g "when submitting a non-alternative currency"